src/formio/components/addressNL.ts

```diff
@@ -19,22 +18,24 @@ export interface AddressComponents {
   city?: HasValidation<'pattern', false>;
 }
 
-export type AddressNLInputSchema = InputComponentSchema<AddressData, Validator, TranslatableKeys>;
+export interface AddressNLExtensions {
+  components?: AddressComponents;
+}
```

Comment:
Nice, it looks a lot better to not have to manually specify the other keys under the `openForms` key :+1: